Checkpoint3/Program.cs

```diff
@@ -0,0 +1,710 @@
+﻿using System;
```

Comment:
Using DisplayId seems to be adding more complexity than its worth. Especially because you can have 2 valid items with the same DisplayId.

Also the display id is not saved to the database consistently.